src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Longer-running asynchronous NRQL queries"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+If you want to run NRQL queries that have a 10-minute [query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries), you can use [our NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to run an asynchronous query. This is useful for queries of a large data set or that query over a long time range. It can also be useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. 
+
+## Requirements [#requirements]
+
+Requires [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-plus). 
+
+## Asynchronous query example [#example]
+
+Here's an example of an asynchronous query NerdGraph call. The `async: true` is what makes the query asynchronous. This query will return results if it completes within the default timeout of 5 seconds; otherwise it will return queryProgress data for use in follow-up queries to the `actor.account.nrqlQueryProgress` field. 
```

Comment:
Probably want to put `queryProgress` in backticks since it's the name of a field.